src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -65,6 +65,7 @@
     <PackageReference Include="System.Reactive.Linq" Version="3.1.1" />
     <PackageReference Include="System.Reactive.PlatformServices" Version="3.1.1" />
     <PackageReference Include="System.Reflection.Emit" Version="4.3.0" />
+    <PackageReference Include="WindowsAzure.Storage" Version="9.3.1" />
```

Comment:
Note I had to add this because `ScriptFunctionMetadataResolver` still needs it. Previous iterations hadn't updated Logging.ApplicationInsights, so there was still a transitive dependency on this. Once I updated that, this failed to build.

Reply:
That makes sense.